### PR TITLE
added shells for Identity classes

### PR DESCRIPTION
**Pull request by BadikVV**
and added "Kirel" prefix for all classes 

Signed-off-by: Vlad Osminin <vladis.osminin@gmail.com>

### CODE REVIEW

File: src/Kirel.Identity.Core/Services/KirelUserService.cs [hunk 1]
```diff
@@ -57,7 +58,7 @@ public class KirelUserService<TKey, TUser, TUserDto, TUserCreateDto, TUserUpdate
     /// <param name="roleManager">Identity role manager</param>
     /// <param name="context">Identity database context</param>
     /// <param name="mapper">AutoMapper instance</param>
-    public KirelUserService(UserManager<TUser> userManager, RoleManager<IdentityRole<TKey>> roleManager, IdentityContext<TKey, TUser> context, IMapper mapper)
+    public KirelUserService(UserManager<TUser> userManager, RoleManager<IdentityRole<TKey>> roleManager, KirelIdentityContext<TKey, TUser> context, IMapper mapper)
```
Comment (Insei) [inline src/Kirel.Identity.Core/Services/KirelUserService.cs]: RoleManager<IdentityRole<TKey>> - please replace with Kirel too.

**Review by Insei — CHANGES_REQUESTED** [review]

File: src/Kirel.Identity.Core/Services/KirelUserService.cs (hunk 1 above)
Comment (BadikVV) [reply src/Kirel.Identity.Core/Services/KirelUserService.cs]: Fixed

File: src/Kirel.Identity.Controllers/KirelUsersController.cs [hunk 2]
```diff
@@ -22,7 +23,7 @@ namespace Kirel.Identity.Controllers;
 public class KirelUsersController<TUserService,TKey,TUser,TUserDto,TUserCreateDto,TUserUpdateDto,TClaimDto,TClaimCreateDto,TClaimUpdateDto> : Controller
     where TUserService : KirelUserService<TKey,TUser,TUserDto,TUserCreateDto,TUserUpdateDto,TClaimDto,TClaimCreateDto,TClaimUpdateDto>
     where TKey : IComparable, IComparable<TKey>, IEquatable<TKey> 
-    where TUser : IdentityUser<TKey>, IKirelUser<TKey> 
+    where TUser : KirelIdentityUser<TKey>, IKirelUser<TKey>
```
Comment (Insei) [inline src/Kirel.Identity.Controllers/KirelUsersController.cs]: Make KirelIdentityUser implement the IKirelUser interface itself and then remove IKirelUser from here

File: src/Kirel.Identity.Core/Context/KirelIdentityContext.cs [hunk 3]
```diff
@@ -8,16 +8,16 @@ namespace Kirel.Identity.Core.Context;
 /// <summary>
 /// Class for the Identity Framework database context used for Guid identity.
 /// </summary>
-public class IdentityContext<TKey, TUser> : IdentityDbContext<TUser, IdentityRole<TKey>, TKey, 
-    IdentityUserClaim<TKey>,IdentityUserRole<TKey>,IdentityUserLogin<TKey>,IdentityRoleClaim<TKey>,IdentityUserToken<TKey>> 
-    where TUser : IdentityUser<TKey>, IKirelUser<TKey> 
+public class KirelIdentityContext<TKey, TUser> : IdentityDbContext<TUser, KirelIdentityRole<TKey>, TKey, 
+    KirelIdentityUserClaim<TKey>,KirelIdentityUserRole<TKey>,KirelIdentityUserLogin<TKey>,KirelIdentityRoleClaim<TKey>,KirelIdentityUserToken<TKey>> 
+    where TUser : KirelIdentityUser<TKey>, IKirelUser<TKey>
```
Comment (Insei) [inline src/Kirel.Identity.Core/Context/KirelIdentityContext.cs]: Same with KirelIdentityUser<TKey>

File: src/Kirel.Identity.Controllers/KirelAuthenticationController.cs [hunk 4]
```diff
@@ -18,7 +19,7 @@ namespace Kirel.Identity.Controllers;
 public class KirelAuthenticationController<TAuthenticationService, TKey, TUser, TRegisterDto> : Controller
     where TAuthenticationService : KirelAuthenticationService<TKey, TUser, TRegisterDto> 
     where TKey : IComparable, IComparable<TKey>, IEquatable<TKey> 
-    where TUser : IdentityUser<TKey>, IKirelUser<TKey> 
+    where TUser : KirelIdentityUser<TKey>, IKirelUser<TKey>
```
Comment (Insei) [inline src/Kirel.Identity.Controllers/KirelAuthenticationController.cs]: same

File: src/Kirel.Identity.Controllers/KirelAuthorizedUserController.cs [hunk 5]
```diff
@@ -19,7 +20,7 @@ namespace Kirel.Identity.Controllers;
 public class KirelAuthorizedUserController<TAuthorizedUserService, TKey, TUser, TAuthorizedUserDto, TAuthorizedUserUpdateDto> : Controller
     where TAuthorizedUserService : KirelAuthorizedUserService<TKey, TUser, TAuthorizedUserDto, TAuthorizedUserUpdateDto>
     where TKey : IComparable, IComparable<TKey>, IEquatable<TKey>
-    where TUser : IdentityUser<TKey>, IKirelUser<TKey>
+    where TUser : KirelIdentityUser<TKey>, IKirelUser<TKey>
```
Comment (Insei) [inline src/Kirel.Identity.Controllers/KirelAuthorizedUserController.cs]: Same.

**Review by Insei — CHANGES_REQUESTED** [review]

File: src/Kirel.Identity.Core/Context/KirelIdentityContext.cs (hunk 3 above)
Comment (BadikVV) [reply src/Kirel.Identity.Core/Context/KirelIdentityContext.cs]: Fixed

File: src/Kirel.Identity.Controllers/KirelUsersController.cs (hunk 2 above)
Comment (BadikVV) [reply src/Kirel.Identity.Controllers/KirelUsersController.cs]: Fixed

File: src/Kirel.Identity.Controllers/KirelAuthenticationController.cs (hunk 4 above)
Comment (BadikVV) [reply src/Kirel.Identity.Controllers/KirelAuthenticationController.cs]: Fixed

File: src/Kirel.Identity.Controllers/KirelAuthorizedUserController.cs (hunk 5 above)
Comment (BadikVV) [reply src/Kirel.Identity.Controllers/KirelAuthorizedUserController.cs]: Fixed

**Review by Insei — APPROVED** [review]

**Pull request merged**